syntax = "proto2";
package oneflow;

message ModelConfig {
  required string name = 1;
  required string saved_model_path = 2;
  optional string saved_model_meta_file_name = 3 [default = "saved_model.prototxt"];
  repeated GraphConfig graphs = 4;
  optional string device_tag = 5;
  optional int32 device_num = 6;
}

message GraphConfig {
  required string graph_name = 1;
  optional string signature_name = 2;
}

message ModelConfigList {
  repeated ModelConfig configs = 1;
}
